extra-network/cgproxy/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=cgproxy
+PKGSEC=net
+PKGDEP="libbpf iproute2 which iptables"
+BUILDDEP="nlohmann-json"
+PKGDES="Transparent proxy with cgroup v2 and tproxy."
```

Comment:
For AOSC OS, never end PKGDES with a period. Also, this description makes no sense.